subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -394,18 +411,41 @@ static void ticker_by_next_slot_get(struct ticker_instance *instance,
 		_ticker_id_head = ticker->next;
 	}
 
+	/* Find first ticker node with match or slot ticks */
+	while (_ticker_id_head != TICKER_NULL) {
+		ticker = &node[_ticker_id_head];
+
+		if (fp_match_op_func) {
```

Comment:
Changes in this PR here, adds this additional branching instructions when `CONFIG_BT_TICKER_SLOT_AGNOSTIC`  and `CONFIG_BT_TICKER_NEXT_SLOT_GET_MATCH` are disabled. Add conditional compile here, as compiler/linker would not optimize this branching (based on static constant parameters if any like `fp_match_op_func`). Observed by building upstream/main peripheral_hr sample with and without this PR (149812 B with this PR vs 149796 B without this PR).